src/MeshVolume.js

```diff
@@ -147,30 +155,11 @@ export default class MeshVolume {
     }
 
     // find the current isosurface opacity and color.
-    let opacity = 1;
-    let color = new Color();
-    if (this.meshrep[channel].material) {
-      opacity = this.meshrep[channel].material.opacity;
-      color = this.meshrep[channel].material.color;
-    } else {
-      this.meshrep[channel].traverse(function(child) {
-        if (child instanceof Mesh) {
-          opacity = child.material.opacity;
-          color = child.material.color;
-        }
-      });
-    }
+    let opacity = this.channel_opacities[channel];
+    let color = this.channel_colors[channel];
```

Comment:
this is real nit but here and below you're using snake_case in javascript files. In other repos I think our eslint rules are set up to enforce camelCase but I guess not here?

Reply:
yup, I noticed this too, I've been in python land for too long. The code is due for a big typescript conversion at some point soon.  Probably at the time of the next big feature addition, I will take a pause and start adding conformance with our other more modern js/ts code.